.github/workflows/grass-manual.yml

```diff
@@ -0,0 +1,45 @@
+name: GRASS GIS addon manual
+
+on:
+  workflow_call:
+
```

Comment:
Maybe add that this should only run for `main` branch?

Reply:
@mmacata Would you write this in the reusable workflow or rather in the other one where you call this one?

Reply:
In the other one would make it more freely usable. But then it would be great if the `main` branch would be the default example in the README